AK/UFixedBigInt.h

```diff
@@ -17,480 +19,420 @@
+
+    DEFINE_COMPARISON_OPERATOR(==, true)
+    DEFINE_COMPARISON_OPERATOR(!=, true)
+    DEFINE_COMPARISON_OPERATOR(<, false)
+    DEFINE_COMPARISON_OPERATOR(>, false)
+    DEFINE_COMPARISON_OPERATOR(<=, false)
+    DEFINE_COMPARISON_OPERATOR(>=, false)
+
+#undef DEFINE_COMPARISON_OPERATOR
+
+#define DEFINE_STANDARD_BINARY_OPERATOR(op, function)                        \
+    constexpr auto operator op(UFixedInt auto const& other) const            \
+    {                                                                        \
+        auto func = [](auto&& a, auto&& b, auto&& c) { function(a, b, c); }; \
+        return do_standard_binary_operation(other, func);                    \
+    }                                                                        \
+                                                                             \
+    constexpr auto operator op(IntegerWrapper other) const                   \
+    {                                                                        \
+        auto func = [](auto&& a, auto&& b, auto&& c) { function(a, b, c); }; \
+        return do_standard_binary_operation(other, func);                    \
+    }
+
+#define DEFINE_STANDARD_COMPOUND_ASSIGNMENT(op, function)                    \
+    constexpr auto& operator op(UFixedInt auto const& other)                 \
+    {                                                                        \
+        auto func = [](auto&& a, auto&& b, auto&& c) { function(a, b, c); }; \
+        do_standard_compound_assignment(other, func);                        \
+        return *this;                                                        \
+    }                                                                        \
+                                                                             \
+    constexpr auto& operator op(IntegerWrapper other)                        \
+    {                                                                        \
+        auto func = [](auto&& a, auto&& b, auto&& c) { function(a, b, c); }; \
+        do_standard_compound_assignment(other, func);                        \
+        return *this;                                                        \
+    }
+
+    // Binary operators
+    DEFINE_STANDARD_BINARY_OPERATOR(^, storage_compute_bitwise<BitwiseOperation::XOR>)
+    DEFINE_STANDARD_BINARY_OPERATOR(&, storage_compute_bitwise<BitwiseOperation::AND>)
+    DEFINE_STANDARD_BINARY_OPERATOR(|, storage_compute_bitwise<BitwiseOperation::OR>)
+    DEFINE_STANDARD_COMPOUND_ASSIGNMENT(^=, storage_compute_inplace_bitwise<BitwiseOperation::XOR>)
+    DEFINE_STANDARD_COMPOUND_ASSIGNMENT(&=, storage_compute_inplace_bitwise<BitwiseOperation::AND>)
+    DEFINE_STANDARD_COMPOUND_ASSIGNMENT(|=, storage_compute_inplace_bitwise<BitwiseOperation::OR>)
+
+    constexpr auto operator~() const
+    {
+        UFixedBigInt<bit_size> result;
+        storage_compute_bitwise<BitwiseOperation::INVERT>(m_data, m_data, result.m_data);
```

Comment:
With taking the previous notes into account,
why not `storage_invert`?

Reply:
Because they are not the same (see above)